docker/Dockerfile

```diff
@@ -3,8 +3,9 @@ FROM node:16
 WORKDIR /automatisch
 
 RUN apt-get update && apt-get install -y postgresql-client
-
-COPY ./wait-for-postgres.sh /automatisch/wait-for-postgres.sh
+COPY ./entrypoint.sh /entrypoint.sh
 
 RUN yarn global add @automatisch/cli@0.2.0
-CMD sh /automatisch/wait-for-postgres.sh automatisch start-worker --env-file /automatisch/storage/.env
+
+EXPOSE 3000
```

Comment:
Since the new dockerfile works for both the main and the worker, I don't think we need to expose the 3000 port for the worker.

Reply:
I believe the `EXPOSE` instruction here does not actually expose the port but acts as a form of documentation. Since we cover the main service functionality here, it is fine to leave it as-is, IMO.

ref: https://docs.docker.com/engine/reference/builder/#expose

> The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published.

Reply:
@barinali, thanks for the info. Then I'm marking this as resolved.